content/backup-restore-docker.rst

```diff
@@ -46,6 +46,18 @@ This means that the webserver and the OTOBO daemon may, but don't have to, be st
     # check the backup file
     docker_admin>tree otobo_backup
 
+.. note::
+
+   To restore the database make sure that the database ``otobo`` exists and contains no tables.
+
+To drop an existing otobo database and create a new one you can use the following commands.
+
+.. code-block:: bash
+
+   docker_admin>docker exec -i otobo_db_1 mysql -u root -p<your_secret_password> -e "DROP DATABASE otobo"
```

Comment:
Hi Andreas,
thanks for the contribution.
Could you change the commands to 'docker-compose exec db' ? We try to avoid the explicit container names like otobo_db_1, as we had customers that ran multiple OTOBO installation on the same Docker host. In that case a `cd /opt/otobo-docker` might also have to be added.

Reply:
To be honest, I tried for about an hour to make a 'nice' example using compose exec and use the root password env var from within the db container, but I unfortunately didn't find any working solution. :( 
That's why I decided to stick with the direct access example, because it should be better than nothing.